### PR TITLE
Allow version 2 of the Doctrine bundle

### PR DESCRIPTION
**Pull request by bytehead**
| Q                | A
| -----------------| ---
| Fixed issues     | n/a
| Docs PR or issue | https://github.com/contao/docs/issues/593

When using Symfony ^4.4 from https://docs.contao.org/dev/getting-started/initial-setup/symfony-application/contao-4.9/, newer doctrine/* versions will be initially installed by `$ composer create-project symfony/website-skeleton contao-example ^4.4`.

This PR opens the constraints for these bundles.

**ToDo:**
- [ ] Update docs https://docs.contao.org/dev/getting-started/initial-setup/symfony-application/contao-4.9/

### CODE REVIEW

**Comment by leofeyer** [comment]
I think you have to adopt the changes from 3651d69e96e93b5959749a7ec981ec6a7f527da8 to make this work.

**Comment by bytehead** [comment]
I've updated accordingly with the backported changes from 3651d69.

File: core-bundle/src/Repository/CronJobRepository.php [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 use Contao\CoreBundle\Entity\CronJob;
 use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
-use Doctrine\Common\Persistence\ManagerRegistry;
 use Doctrine\DBAL\Connection;
+use Symfony\Bridge\Doctrine\ManagerRegistry;
```
Comment (leofeyer) [inline core-bundle/src/Repository/CronJobRepository.php:18]: How is that better than using `Doctrine\Persistence\ManagerRegistry`?

Comment (bytehead) [reply core-bundle/src/Repository/CronJobRepository.php:18]: Because it won't work over all versions otherwise (see c4612fe).

Comment (bytehead) [reply core-bundle/src/Repository/CronJobRepository.php:18]: The functional tests on `prefer-lowest` do fail: https://github.com/contao/contao/pull/2321/checks?check_run_id=1301899600

**Review by ausi — APPROVED** [review]

File: core-bundle/src/Repository/CronJobRepository.php (hunk 1 above)
Comment (leofeyer) [reply core-bundle/src/Repository/CronJobRepository.php:18]: I see. 👍🏻

**Comment by leofeyer** [comment]
Thank you @bytehead.

**Pull request merged**